models/data_warehouse.model.lkml

```diff
@@ -3105,19 +3105,20 @@ explore: license_server_fact {
   }
 
   join: server_daily_details {
-    sql_on: ${server_fact.server_id} = ${server_daily_details_ext.server_id} ;;
+    sql_on: ${server_fact.server_id} = ${server_daily_details.server_id} ;;
```

Comment:
@sri-byte , why having `server_fact` and not `license_server_fact` here?

Reply:
Also, which ticket link does it refer to?

Reply:
Hi @catalintomai, server_fact(server telemetry) has more servers/data than license_server_fact (license telemetry), hence the join on server_fact MM-50498

Reply:
Thanks, @sri-byte , if it is a superset, should we open a ticket to scrub references to `license_server_fact ` and replace them with `server_fact `, where applicable?  
Also, please update the "Ticket Link" value in the  description with the link to MM-50498. That is the format other engineering teams are using (see mattermost-server PRs foe example). Thanks!

Reply:
We cannot replace, they both are different sources of telemetry one from licenses and other from servers, both are required. Licenses captures data that server telemetry doesn't/cannot capture. Added the ticket link to the description.